Car.cs

```diff
@@ -0,0 +1,30 @@
+namespace CarApplication
+{
+    public class Car
+    {
+        public static void DrivingCar()
+        {
+            Random randomNumber = new Random();
+            int chanceOfUncertainity = randomNumber.Next(1, 10);
```

Comment:
can have a more meaningful name rather than **chanceOfUncertainity**

Reply:
Can we go with **RandomEvent**?

Reply:
How about **randomDrivingEvent**? It seems a bit more specific.
**randomEvent** also works.